luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -0,0 +1,241 @@
+import hail as hl
+
+from lib.model.base_mt_schema import BaseMTSchema, row_annotation, RowAnnotationOmit
+from lib.model.seqr_mt_schema import SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+
+from hail_scripts.computed_fields import variant_id
+
+
+BOTHSIDES_SUPPORT = "BOTHSIDES_SUPPORT"
+PASS = "PASS"
+
+# Used to filter mt.info fields.
+CONSEQ_PREDICTED_PREFIX = 'PREDICTED_'
+NON_GENE_PREDICTIONS = {'PREDICTED_INTERGENIC', 'PREDICTED_NONCODING_BREAKPOINT', 'PREDICTED_NONCODING_SPAN'}
+
+
+INTERVAL_TYPE = 'array<struct{type: str, chrom: str, start: int32, end: int32}>'
+
+
+def get_cpx_interval(x):
+    # an example format of CPX_INTERVALS is "DUP_chr1:1499897-1499974"
+    type_chr = x.split('_chr')
+    chr_pos = type_chr[1].split(':')
+    pos = chr_pos[1].split('-')
+    return hl.struct(type=type_chr[0], chrom=chr_pos[0], start=hl.int32(pos[0]), end=hl.int32(pos[1]))
+
+
+class SeqrSVVariantSchema(BaseMTSchema):
+
+    def __init__(self, *args, gene_id_mapping, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.gene_id_mapping = gene_id_mapping
+
+    @row_annotation()
+    def contig(self):
+        # 'contig': lambda rows: rows.locus.contig.replace('^chr', ''),
+        return variant_id.get_expr_for_contig(self.mt.locus)
+
+    @row_annotation()
+    def sc(self):
+        return self.mt.info.AC[0]
+
+    @row_annotation()
+    def sf(self):
+        return self.mt.info.AF[0]
+
+    @row_annotation()
+    def sn(self):
+        return self.mt.info.AN
+
+    @row_annotation()
+    def start(self):
+        # 'start': lambda rows: rows.locus.position,
+        return variant_id.get_expr_for_start_pos(self.mt)
+
+    @row_annotation()
+    def end(self):
```

Comment:
This deviates from the seqr_mt_schema which uses:

```
@row_annotation(disable_index=True)
    def end(self):
        return variant_id.get_expr_for_end_pos(self.mt)
```

just curious about why?